src/components/Modal.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+const Modal = ({ show, modalTitle, description, close }) => {
+  return (
+    <>
+      {show ? (
+        <div className="modalContainer" onClick={() => close()}>
```

Comment:
there is no need to assign a new function, if  params are not needed to be transformed.

Reply:
```
        <div className="modalContainer" onClick={close}>
```

Reply:
done